burnman/composite.py

```diff
@@ -33,63 +37,144 @@ class Composite(Material):
 
     This class is available as ``burnman.Composite``.
     """
-    def __init__(self, fractions, phases=None):
+    def __init__(self, phases, fractions=None, fraction_type='molar'):
         """
         Create a composite using a list of phases and their fractions (adding to 1.0).
 
         Parameters
         ----------
-        fractions: list of floats
-            molar fraction for each phase.
         phases: list of :class:`burnman.Material`
             list of phases.
+        fractions: list of floats
+            molar fraction for each phase.
```

Comment:
also the documentation of "fraction" is no longer correct (depends on fraction_type)